cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -89,7 +93,8 @@ func newPlugin(tokenProvider tokenProvider) *plugin {
 var (
 	useApplicationDefaultCredentials = pflag.Bool("use_application_default_credentials", false, "Output is an ExecCredential filled with application default credentials.")
 	useEdgeCloud                     = pflag.Bool("use_edge_cloud", false, "Output is an ExecCredential for an Edge Cloud cluster.")
-	project                          = pflag.String("project", "", fmt.Sprintf("Parent project of the Cluster %s.", useEdgeCloudOnlyMessage))
+	project                          = pflag.String("project", "", fmt.Sprintf("Parent project of the Cluster %s.", useEdgeCloudOnlyRequiredMessage))
```

Comment:
I think we can remove "useEdgeCloudOnlyRequiredMessage" or "useEdgeCloudOnlyMessage" as we can use it in multiple places, right? Having that "edge cloud" only is not really relevant..

Reply:
The case for having 2 is for the most clear messaging to the conumer, in the case for a flag like `location` it is only applicable when using the edge cloud flag. It is also required though, so technically it falls under both messages. I can't think of some clear verbiage that can cover all cases in once scentence, since we have the cases of flags that are applicable only to edge_cloud, applicable only to edge_cloud AND required by it, applicable to both edge_cloud and non-edge_cloud BUT required in edge cloud.

The message I chose was one that distinguished between only applicable to edge_cloud and one that can be used outside of edge_cloud but also required to use the edge_cloud flag.

Reply:
I see what you mean.. Can we update the useEdgeCloudRequiredMessage = "mandatory when using --use_edge_cloud, optional otherwise"